code/bmpman/bmpman.cpp

```diff
@@ -68,8 +69,8 @@ const BM_TYPE bm_type_list[] = { BM_TYPE_DDS, BM_TYPE_TGA, BM_TYPE_PNG, BM_TYPE_
 const char *bm_ext_list[] = { ".dds", ".tga", ".png", ".jpg", ".pcx" };
 const int BM_NUM_TYPES = sizeof(bm_type_list) / sizeof(bm_type_list[0]);
 
-const BM_TYPE bm_ani_type_list[] = { BM_TYPE_EFF, BM_TYPE_ANI };
-const char *bm_ani_ext_list[] = { ".eff", ".ani" };
+const BM_TYPE bm_ani_type_list[] = { BM_TYPE_EFF, BM_TYPE_ANI, BM_TYPE_PNG };
+const char *bm_ani_ext_list[] = { ".eff", ".ani", ".png" };
```

Comment:
From a reasonably trivial search through the code I have identified a number of areas where it is assumed that file extensions are 3 chars long. And to be sure you get all of them it'll probably require checking every single instance where filenames are handled :s 

Therefore I think I'll leave the extension as .png. Given that I've fixed the issue further up about leaking cfile handles I don't think there's a specific issue that using .apng as the extension will solve.

Additionally, the PR as is will allow an apng to be loaded as a single image if required. e.g. calls to bm_load on an apng will get the default image as intended by the apng spec. This could reduce the amount of images required to be shipped with a mod (note that the default image doesn't have to be part of the sequence of animation images). 
I've current disallowed the reverse, i.e. animations to load a png as a single frame animation because it's unknown how long such an "animation" should be. Also, apngasm refuses to create a single frame apng; they figure that you've already got what you want in the static png image.

Anyway, lots of ~words~, any comments?